app/src/main/java/gov/hhs/cdc/trustedintermediary/etor/order/OrderController.java

```diff
@@ -8,13 +14,57 @@ public class OrderController {
 
     private static final OrderController ORDER_CONTROLLER = new OrderController();
 
+    private Formatter jsonFormatter = ApplicationContext.getImplementation(Formatter.class);
```

Comment:
```suggestion
    @Inject Formatter formatter;
```
It also shouldn't be called a `jsonFormatter` because we are never guaranteed that the `Formatter`'s implementation will be for JSON.

Reply:
I like that. I'm on it.